src/Jackett/Services/IndexerManagerService.cs

```diff
@@ -194,6 +194,8 @@ private string GetIndexerConfigFilePath(IIndexer indexer)
 
         public void SaveConfig(IIndexer indexer, JToken obj)
         {
+            indexer.LoadFromSavedConfiguration(obj); // This seems crazy, but without this call the indexer will represent itself as unconfigured (which is misleading).
```

Comment:
I changed the isConfigured/SaveConfig order with 6b35da8ad0fb0048052f07041ed12484b4e6bf06.
This should make it more consistent. Some indexers were already doing it in this order.
Certainly not the best option but I prefer it over loading the configuration on each save call (which might happen a lot in some situations).